### DISCUSSION TITLE
Lost account

### DISCUSSION THREAD

**Discussion by A6Leen** [community/community#32933] [Other Feature Feedback, Questions, & Ideas] [answered]
so I have a problem I have had an account a long time ago but when I wanted to sign in it said to reset the password because it was weak..etc I am not sure what happened but I created another account with the same email for the old one so every time I want to get to my old profile it login me into this one I am not sure how to fix this

**Comment by radar-blip** [comment, marked as answer]
For account related issues, try here: [Support Page](https://support.github.com/contact/account)

**Reply by queenofcorgis** [reply]
@linuxlawson has got it right, the best place to go for answers to account-specific questions like this one is by opening a ticket on our [Support page](https://support.github.com/contact).

**Reply by A6Leen** [reply]
thxx
